bigchaindb/core.py

```diff
@@ -410,7 +347,7 @@ def get_owned_ids(self, owner):
 
     @property
     def fastquery(self):
```

Comment:
This method can be removed in favour of [this](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/tendermint/lib.py#L346)